code/game/gamemodes/miniantags/abduction/machinery/console.dm

```diff
@@ -29,7 +29,7 @@
 	var/list/datum/icon_snapshot/disguises = list()
 
 /obj/machinery/abductor/console/Initialize()
-	..()
+	. = ..()
 	return INITIALIZE_HINT_LATELOAD
```

Comment:
This is why you dont do mass find/replace.

This call was fine, as it returns a hint right underneath it, and the setting of `.` incurs extra instruction overhead.